docs/source/amp.rst

```diff
@@ -256,6 +256,141 @@ In this case, combine the two layers using :func:`torch.nn.functional.binary_cro
 or :mod:`torch.nn.BCEWithLogitsLoss`.  ``binary_cross_entropy_with_logits`` and ``BCEWithLogits``
 are safe to autocast.
 
+.. _autocast-xpu-op-reference:
+
+XPU Op-Specific Behavior
+------------------------
+The following lists describe the behavior of eligible ops in autocast-enabled regions.
+These ops always go through autocasting whether they are invoked as part of a :class:`torch.nn.Module`,
+as a function, or as a :class:`torch.Tensor` method. If functions are exposed in multiple namespaces,
+they go through autocasting regardless of the namespace.
+
+Ops not listed below do not go through autocasting.  They run in the type
+defined by their inputs.  However, autocasting may still change the type
+in which unlisted ops run if they're downstream from autocasted ops.
+
+If an op is unlisted, we assume it's numerically stable in ``bfloat16``.
+If you believe an unlisted op is numerically unstable in ``bfloat16``,
+please file an issue.
+
+XPU Ops that can autocast to ``bfloat16``
+""""""""""""""""""""""""""""""""""""""""
+
+``__matmul__``,
+``addbmm``,
+``addmm``,
+``addmv``,
+``addr``,
+``baddbmm``,
+``bmm``,
+``chain_matmul``,
+``multi_dot``,
+``conv1d``,
+``conv2d``,
+``conv3d``,
+``conv_transpose1d``,
+``conv_transpose2d``,
+``conv_transpose3d``,
+``GRUCell``,
+``linear``,
+``LSTMCell``,
+``matmul``,
+``mm``,
+``mv``,
+``prelu``,
```

Comment:
Not currently supported by XPU [1]. I did not check whether other items in this list are supported by XPU backend - see comment below.

[1] https://github.com/intel/torch-xpu-ops/blob/65c61554aa1023ee99dce101dc743317d705b863/src/aten/XPUFallback.template#L351